backend/src/scripts/utility/index.js

```diff
@@ -59,7 +61,15 @@ const parser = populateCommands(Yargs(hideBin(process.argv)))
 	.scriptName('yarn run-script')
 	.wrap(Yargs().terminalWidth())
 	.demandCommand()
+	.fail((msg, err) => {
+		if (msg) {
+			Yargs().showHelp();
+		}
+		handleErrorBeforeExit(msg ?? err);
+	})
 	.parse();
 
-// eslint-disable-next-line no-console
-Promise.resolve(parser).catch(console.error).finally(process.exit);
+Promise.resolve(parser).catch(handleErrorBeforeExit).finally(() => {
+	// eslint-disable-next-line no-process-exit
+	process.exit();
+});
```

Comment:
@carmenfan Running `yarn run-script` without specifying a script now gives an error:
```
yarn run-script
yarn run v1.22.5
$ cross-env NODE_CONFIG_DIR=../config/remote node ./src/scripts/utility/index
Options:
  --help     Show help                                                 [boolean]
  --version  Show version number                                       [boolean]
2023-03-06T12:42:50.405Z [error] [APP] 	Not enough non-option arguments: got 0, need at least 1 
error Command failed with exit code 1.
```

Reply:
I don't see a problem with this. You didn't give it any command so it didn't know what to do. It shoulds the options so the user knows to `--help` to proceed

@keifgwinn  do you care?

Reply:
yes but not enough to stop this pull request from going through as the other functions are more useful